### DISCUSSION TITLE
Command Line Interface..?

### DISCUSSION THREAD

**Discussion by LordZozzy** [Komet/MediaElch#1428] [Q&A] [answered]
Hey,

I was just looking for a way to automate Mediaelch scraping and came across the changelog for version 2.6.2, which claims the integration of a CLI.
However nowhere can I find any information on using this, can you guys help me?

I'm currently on 2.8.16, Windows.

Thanks!

**Comment by bugwelle** [comment, marked as answer]
Hi,

MediaElch _does_ have a basic command line interface. However, it is _not_ ready yet and is not available through package managers. You will need to manually compile MediaElch. See https://github.com/Komet/MediaElch/issues/77

Regards,
Andre
